instagram/src/components/Comment.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './CommentSection.css';
+
+const Comment = props => {
+  return (
+    <div className="comment-text">
+      <span className="comment">{props.comment.text}</span>{' '}
```

Comment:
Make sure to look at the image in the README to see what display you're trying to emulate. The username should come before the text, so the order should be changed here. And it might be easiest to just do something like `<div><b>{props.comment.username}</b> {props.comment.text}</div>`.